swarm.go

```diff
@@ -68,15 +68,15 @@ var (
 	uptimeGauge        = metrics.NewRegisteredGauge("stack.uptime", nil)
 )
 
-// the swarm stack
+// Swarm is an object abstracting the complete Swarm stack
 type Swarm struct {
 	config            *api.Config        // swarm configuration
 	api               *api.API           // high level api layer (fs/manifest)
 	dns               api.Resolver       // DNS registrar
 	fileStore         *storage.FileStore // distributed preimage archive, the local API to the storage with document level storage/retrieval support
 	streamer          *stream.Registry
-	bzz               *network.Bzz       // the logistic manager
-	backend           chequebook.Backend // simple blockchain Backend
+	bzz               *network.Bzz // the logistic manager
+	backend           cswap.Backend
```

Comment:
`swapBackend`?
a bit more context would be nice
(i know that the original field name is the same. but my request still stands)

Reply:
isn't the context given by the package in this case?

Reply:
ok